tests/test_entities.py

```diff
@@ -431,6 +433,24 @@ def test_no_such_path_error(self):
                 with self.assertRaises(NoSuchPathError):
                     entity(self.cfg_610).path(which=which)
 
+    def test_arfreport(self):
+        """Test :meth:`nailgun.entities.ArfReport.path`.
+        Assert that the following return appropriate paths:
+        * ``ExternalUserGroup(id=…).path()``
+        * ``ExternalUserGroup(id=…).path('download_html')``
```

Comment:
Is ExternalUserGroup correct? Or should this be ArfReport?

Reply:
@JacobCallahan updated. Thanks.